pkg/api/types.go

```diff
@@ -173,9 +173,9 @@ type VolumeSource struct {
 	HostPath *HostPathVolumeSource `json:"hostPath"`
 	// EmptyDir represents a temporary directory that shares a pod's lifetime.
 	EmptyDir *EmptyDirVolumeSource `json:"emptyDir"`
-	// GCEPersistentDisk represents a GCE Disk resource that is attached to a
+	// CloudPersistentDisk represents a cloud-provided storage resource that is attached to a
 	// kubelet's host machine and then exposed to the pod.
-	GCEPersistentDisk *GCEPersistentDiskVolumeSource `json:"persistentDisk"`
+	CloudPersistentDisk *CloudPersistentDiskVolumeSource `json:"persistentDisk"`
```

Comment:
Yeah, I'd started this before I saw that there were different viewpoints on #5129.  Really looking for feedback on the AWS stuff specifically; once we figure out #5129 then I'll rebase the AWS work on to whatever we choose.